src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -479,9 +477,6 @@ public function dumpFile($filename, $content, $mode = 0666)
         }
 
         $this->rename($tmpFile, $filename, true);
-        if (null !== $mode) {
-            $this->chmod($filename, $mode);
```

Comment:
The default value of the `$mode` param is 0666, so the error will be triggered ever